packages/cli/src/cmds/validator/options.ts

```diff
@@ -303,15 +304,16 @@ export const validatorOptions: CliCommandOptions<IValidatorCliArgs> = {
     type: "boolean",
   },
 
-  // Remote signer
+  // External signer
```

Comment:
I tried to make the wording a bit more consistent within the code, and also few logs. We generally use the term "external signer" to describe a external/(remote) server that handles the signing operations for a specific public key. Whereas the term "remote signer" is also used to describe a [remote signer](https://github.com/ChainSafe/lodestar/blob/f5148b2b915f714a77bf790ddf9179aa1e89016a/packages/validator/src/services/validatorStore.ts#L59-L63) in the local validator store.